assets/liquid_staking/bemo.json

```diff
@@ -17,6 +17,36 @@
             ],
             "submittedBy": "shuva10v",
             "submissionTimestamp": "2025-01-17T07:07:33Z"
+        },
+        {
+            "address": "EQAORkhhrXis_XxLAAfyxTyNceNXLtaHcJ5FhXxpNJ8xsDGc",
+            "source": "tonviewer",
+            "comment": "Token that will be exchanged for bemo $BMO token",
+            "tags": [
+                "defi"
+            ],
+            "submittedBy": "rAndom1ze",
+            "submissionTimestamp": "2025-03-12T00:00:00Z"
+        },
+        {
+            "address": "EQA3yiIotfSfBI_xx6xJpXEUnTR8DL6BLUZRVRB7VtvJwrEE",
+            "source": "tonviewer",
+            "comment": "Bemo airdrop wallet sending stXP",
+            "tags": [
+                "defi"
+            ],
+            "submittedBy": "rAndom1ze",
+            "submissionTimestamp": "2025-03-12T00:00:00Z"
+        },
+        {
+            "address": "EQBlzrj9Mhls_QSh9_FUUIzBjytPF3nbYytnavN9ltu5SvFR",
```

Comment:
I see a lot of activity on this wallet not connected to the project directly. Let's remove it